app/ceom/photos/views.py

```diff
@@ -28,6 +27,8 @@
 from ceom.photos.templatetags.photos_tags import thumbnail, point2str
 from ceom.photos.models import Photo, Category, CategoryVote
 from ceom.photos.forms import SearchForm, PhotoForm, BatchEditForm
+from ceom.photos.models import PhotoUser
```

Comment:
You can merge this with a line just above this one:
`from ceom.photos.models import Photo, Category, CategoryVote`
By making it
`from ceom.photos.models import Photo, Category, CategoryVote, PhotoUser`